www/pages/company.tsx

```diff
@@ -26,12 +26,33 @@ import {
   Space,
   Typography,
 } from '@supabase/ui'
+import { NextSeo } from 'next-seo'
 
 type Props = {}
 
 const Index = ({}: Props) => {
+  // base path for images
+  const { basePath } = useRouter()
+
+  const meta_title = 'Company | Supabase'
+  const meta_description = 'See how the supabase runs.'
```

Comment:
const meta_title = 'One of the world's fastest-growing open source communities | Supabase'
const meta_title = 'Supabase is the community that builds the infrastructure for your applications. Build using Supabase for any size project—from a new startup to huge public companies'